### PR TITLE
New keepscussion participants endpoints + ui

### PR DESCRIPTION
**Pull request by cvializ**
@andrewconner @eishay after this is reviewed I'll send a QA build out

### CODE REVIEW

File: packrat/scripts/message_participants.js [hunk 1]
```diff
@@ -598,4 +609,33 @@ k.messageParticipants = k.messageParticipants || (function ($, win) {
...
+      return 0;
+    }
+  }
+
+  function idIs(id) {
+    return function (o) { return o.id === id; };
+  }
+
+  function isLibrary(o) {
+    return o && o.isLibrary === true || o.kind === 'library';
```
Comment (ryanpbrewster) [inline packrat/scripts/message_participants.js]: Do you need parens here? `return o && (o.isLibrary || o.kind === 'library');`

Comment (cvializ) [inline packrat/scripts/message_participants.js]: mhm you're right

**Comment by ryanpbrewster** [comment]
Jesus christ you're a wizard @cvializ

**Comment by cvializ** [comment]
merging to :ship:

**Pull request merged**